src/lightning/pytorch/loggers/wandb.py

```diff
@@ -634,6 +634,28 @@ def finalize(self, status: str) -> None:
         if self._checkpoint_callback and self._experiment is not None:
             self._scan_and_log_checkpoints(self._checkpoint_callback)
 
+    def on_log_checkpoint_artifact(
+        self,
+        artifact: "wandb.Artifact",
+        checkpoint_timestamp: float,
+        path: _PATH,
+        score: float,
+        tag: str,
+    ) -> None:
+        """Override this to provide custom artifact logging behavior.
+
+        By default, adds `path` as a file called "model.ckpt" to the `artifact`.
+
+        Args:
+            artifact (wandb.Artifact): The wandb artifact for this checkpoint
+            checkpoint_timestamp (float): The timestamp the checkpoint was last created or modified.
+            path (_PATH): The absolute path or URI to the checkpoint.
+            score (float): The score associated with the checkpoint.
+            tag (str): The tag associated with the checkpoint.
```

Comment:
```suggestion
            artifact: The wandb artifact for this checkpoint
            checkpoint_timestamp: The timestamp the checkpoint was last created or modified.
            path: The absolute path or URI to the checkpoint.
            score: The score associated with the checkpoint.
            tag: The tag associated with the checkpoint.
```
this is already sourced from typing